### PR TITLE
ESLint action (reviewdog) test: break Wine script

### PR DESCRIPTION
**Pull request by plata**

### CODE REVIEW

**Comment by plata** [comment]
This is quite nice but it didn't find all issues.

**Comment by Zemogiter** [comment]
Do we really need this? I mean Codacy is helpful/anoying as it is.

**Comment by plata** [comment]
@Zemogiter I think it might be easier for people if they see the issues directly in the PR.

**Comment by madoar** [comment]
@plata I don't think that it didn't find all issues. Codacy found the same number of issues. Perhaps the other findings are not marked as issues?

**Comment by madoar** [comment]
If you look at Codacy you will see an additional section on the issues page called "possible new issues". These are not shown in the Codacy statistics and also not by reviewdog

**Comment by plata** [comment]
I don't know. What's possible suppossed

**Comment by madoar** [comment]
I just discovered that Codacy apparently is also able to create such comments, see https://support.codacy.com/hc/en-us/articles/207280219-GitHub-Integration.
I just enabled that flag to see whether it will lead to some useful comments.

File: Engines/Wine/Engine/Object/script.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 const WineEngine = include("engines.wine.engine.implementation");
-const { fileExists } = include("utils.functions.filesystem.files");
+const { fileExists} = include("utils.functions.filesystem.files");
```
Comment (plata) [inline Engines/Wine/Engine/Object/script.js]: Codacy Issue found: [A space is required before '}'.](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4765301)

File: Engines/Wine/Engine/Object/script.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ const ProcessBuilderClass = Java.type("java.lang.ProcessBuilder");
 // eslint-disable-next-line no-unused-vars
 module.default = class Wine {
     constructor() {
-        this._implementation = new WineEngine();
+                 this._implementation = new WineEngine();
```
Comment (plata) [inline Engines/Wine/Engine/Object/script.js]: Codacy Issue found: [Expected indentation of 8 spaces but found 17.](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4765301)

**Comment by madoar** [comment]
I think Codacy is unable to comment on these "possible issues". The hint field for "possible issues" says:

> These issues were introduced in your commit but outside the scope of your changes.

If this is the case you can't add a comment to these issues inside a PR.

Off-topic: It seems like Codacy supports a new integration method for GitHub (maybe we should change to that?).

**Comment by madoar** [comment]
@plata can this PR be closed?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    "description" : "A collection of scripts for Phoenicis",
+    "directories" : {
+        "doc" : "docs"
+    },
+    "scripts" : {
+        "lint"       : "eslint -c .eslintrc.yml ./**/*.js",
+        "json-align" : "find . -name \"*.json\" -print0 | xargs -0 json-align -i"
+    },
+    "repository" : {
+        "type" : "git",
```
Comment (plata) [inline package.json:13]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4765301)

File: package.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    ],
+    "license" : "LPGPL-3.0",
+    "bugs"    : {
+        "url" : "https://github.com/PhoenicisOrg/scripts/issues"
+    },
+    "homepage"        : "https://github.com/PhoenicisOrg/scripts",
+    "devDependencies" : {
+        "eslint"     : "^6.8.0",
+        "json-align" : "^0.1.0"
+    }
```
Comment (plata) [inline package.json:28]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4765301)

File: package.json [hunk 5]
```diff
@@ -0,0 +1,29 @@
+{
+    "name"        : "phoenicis-scripts",
+    "version"     : "1.0.0",
+    "description" : "A collection of scripts for Phoenicis",
+    "directories" : {
```
Comment (plata) [inline package.json:5]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4765301)

File: Engines/Wine/Engine/Object/script.js (hunk 2 above)
Comment (plata) [inline Engines/Wine/Engine/Object/script.js:16]: Codacy Issue found: [Expected indentation of 8 spaces but found 17.](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4765301)

File: package.json [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+        "wine"
+    ],
+    "license" : "LPGPL-3.0",
+    "bugs"    : {
+        "url" : "https://github.com/PhoenicisOrg/scripts/issues"
+    },
+    "homepage"        : "https://github.com/PhoenicisOrg/scripts",
+    "devDependencies" : {
+        "eslint"     : "^6.8.0",
+        "json-align" : "^0.1.0"
```
Comment (plata) [inline package.json:27]: Codacy Issue found: [Format Error: unexpected "    "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4765301)

File: package.json [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+        "json-align" : "find . -name \"*.json\" -print0 | xargs -0 json-align -i"
+    },
+    "repository" : {
+        "type" : "git",
+        "url"  : "git+https://github.com/PhoenicisOrg/scripts.git"
+    },
+    "keywords" : [
+        "phoenicis",
+        "wine"
+    ],
```
Comment (plata) [inline package.json:19]: Codacy Issue found: [Format Error: unexpected " "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4765301)

File: package.json [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+    "version"     : "1.0.0",
+    "description" : "A collection of scripts for Phoenicis",
+    "directories" : {
+        "doc" : "docs"
+    },
+    "scripts" : {
+        "lint"       : "eslint -c .eslintrc.yml ./**/*.js",
+        "json-align" : "find . -name \"*.json\" -print0 | xargs -0 json-align -i"
+    },
+    "repository" : {
```
Comment (plata) [inline package.json:12]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4765301)

File: package.json [hunk 9]
```diff
@@ -0,0 +1,29 @@
+{
+    "name"        : "phoenicis-scripts",
+    "version"     : "1.0.0",
+    "description" : "A collection of scripts for Phoenicis",
+    "directories" : {
+        "doc" : "docs"
+    },
+    "scripts" : {
+        "lint"       : "eslint -c .eslintrc.yml ./**/*.js",
+        "json-align" : "find . -name \"*.json\" -print0 | xargs -0 json-align -i"
```
Comment (plata) [inline package.json:10]: Codacy Issue found: [Format Error: unexpected "    "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4765301)

File: package.json [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+    },
+    "keywords" : [
+        "phoenicis",
+        "wine"
+    ],
+    "license" : "LPGPL-3.0",
+    "bugs"    : {
+        "url" : "https://github.com/PhoenicisOrg/scripts/issues"
+    },
+    "homepage"        : "https://github.com/PhoenicisOrg/scripts",
```
Comment (plata) [inline package.json:24]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4765301)

File: package.json [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+        "type" : "git",
+        "url"  : "git+https://github.com/PhoenicisOrg/scripts.git"
+    },
+    "keywords" : [
+        "phoenicis",
+        "wine"
+    ],
+    "license" : "LPGPL-3.0",
+    "bugs"    : {
+        "url" : "https://github.com/PhoenicisOrg/scripts/issues"
```
Comment (plata) [inline package.json:22]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4765301)

File: package.json [hunk 12]
```diff
@@ -0,0 +1,29 @@
+{
+    "name"        : "phoenicis-scripts",
+    "version"     : "1.0.0",
+    "description" : "A collection of scripts for Phoenicis",
+    "directories" : {
+        "doc" : "docs"
```
Comment (plata) [inline package.json:6]: Codacy Issue found: [Format Error: unexpected " "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4765301)

File: package.json [hunk 13]
```diff
@@ -0,0 +1,29 @@
...
+    },
+    "scripts" : {
+        "lint"       : "eslint -c .eslintrc.yml ./**/*.js",
+        "json-align" : "find . -name \"*.json\" -print0 | xargs -0 json-align -i"
+    },
+    "repository" : {
+        "type" : "git",
+        "url"  : "git+https://github.com/PhoenicisOrg/scripts.git"
```
Comment (plata) [inline package.json:14]: Codacy Issue found: [Format Error: unexpected "    "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4765301)

File: package.json [hunk 14]
```diff
@@ -0,0 +1,29 @@
+{
+    "name"        : "phoenicis-scripts",
+    "version"     : "1.0.0",
+    "description" : "A collection of scripts for Phoenicis",
+    "directories" : {
+        "doc" : "docs"
+    },
+    "scripts" : {
+        "lint"       : "eslint -c .eslintrc.yml ./**/*.js",
```
Comment (plata) [inline package.json:9]: Codacy Issue found: [Format Error: unexpected "    "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4765301)

File: package.json [hunk 15]
```diff
@@ -0,0 +1,29 @@
...
+    },
+    "repository" : {
+        "type" : "git",
+        "url"  : "git+https://github.com/PhoenicisOrg/scripts.git"
+    },
+    "keywords" : [
+        "phoenicis",
+        "wine"
+    ],
+    "license" : "LPGPL-3.0",
```
Comment (plata) [inline package.json:20]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4765301)

File: Engines/Wine/Engine/Object/script.js (hunk 1 above)
Comment (plata) [inline Engines/Wine/Engine/Object/script.js:2]: Codacy Issue found: [A space is required before '}'.](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4765301)

File: package.json [hunk 16]
```diff
@@ -0,0 +1,29 @@
...
+    "scripts" : {
+        "lint"       : "eslint -c .eslintrc.yml ./**/*.js",
+        "json-align" : "find . -name \"*.json\" -print0 | xargs -0 json-align -i"
+    },
+    "repository" : {
+        "type" : "git",
+        "url"  : "git+https://github.com/PhoenicisOrg/scripts.git"
+    },
+    "keywords" : [
+        "phoenicis",
```
Comment (plata) [inline package.json:17]: Codacy Issue found: [Format Error: unexpected " "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4765301)

File: package.json [hunk 17]
```diff
@@ -0,0 +1,29 @@
+{
+    "name"        : "phoenicis-scripts",
+    "version"     : "1.0.0",
+    "description" : "A collection of scripts for Phoenicis",
+    "directories" : {
+        "doc" : "docs"
+    },
```
Comment (plata) [inline package.json:7]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4765301)

File: package.json [hunk 18]
```diff
@@ -0,0 +1,29 @@
...
+    "keywords" : [
+        "phoenicis",
+        "wine"
+    ],
+    "license" : "LPGPL-3.0",
+    "bugs"    : {
+        "url" : "https://github.com/PhoenicisOrg/scripts/issues"
+    },
+    "homepage"        : "https://github.com/PhoenicisOrg/scripts",
+    "devDependencies" : {
```
Comment (plata) [inline package.json:25]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4765301)

File: package.json [hunk 19]
```diff
@@ -0,0 +1,29 @@
+{
+    "name"        : "phoenicis-scripts",
+    "version"     : "1.0.0",
```
Comment (plata) [inline package.json:3]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4765301)

File: package.json [hunk 20]
```diff
@@ -0,0 +1,29 @@
...
+    "description" : "A collection of scripts for Phoenicis",
+    "directories" : {
+        "doc" : "docs"
+    },
+    "scripts" : {
```
Comment (plata) [inline package.json:8]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4765301)

File: package.json [hunk 21]
```diff
@@ -0,0 +1,29 @@
+{
```
Comment (plata) [inline package.json:1]: Codacy Issue found: [JSON is not sorted](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4765301)

File: package.json [hunk 22]
```diff
@@ -0,0 +1,29 @@
...
+    "name"        : "phoenicis-scripts",
+    "version"     : "1.0.0",
+    "description" : "A collection of scripts for Phoenicis",
```
Comment (plata) [inline package.json:4]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4765301)

File: package.json [hunk 23]
```diff
@@ -0,0 +1,29 @@
...
+    "repository" : {
+        "type" : "git",
+        "url"  : "git+https://github.com/PhoenicisOrg/scripts.git"
+    },
+    "keywords" : [
+        "phoenicis",
+        "wine"
+    ],
+    "license" : "LPGPL-3.0",
+    "bugs"    : {
```
Comment (plata) [inline package.json:21]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4765301)

File: package.json [hunk 24]
```diff
@@ -0,0 +1,29 @@
+{
+    "name"        : "phoenicis-scripts",
```
Comment (plata) [inline package.json:2]: Codacy Issue found: [Format Error: unexpected "  "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4765301)

File: package.json [hunk 25]
```diff
@@ -0,0 +1,29 @@
...
+        "phoenicis",
+        "wine"
+    ],
+    "license" : "LPGPL-3.0",
+    "bugs"    : {
+        "url" : "https://github.com/PhoenicisOrg/scripts/issues"
+    },
+    "homepage"        : "https://github.com/PhoenicisOrg/scripts",
+    "devDependencies" : {
+        "eslint"     : "^6.8.0",
```
Comment (plata) [inline package.json:26]: Codacy Issue found: [Format Error: unexpected " "](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4765301)

**Pull request closed without merging**